lib/pbench/server/database/models/templates.py

```diff
@@ -139,7 +137,7 @@ def find(name: str) -> "Template":
         try:
             template = Database.db_session.query(Template).filter_by(name=name).first()
         except SQLAlchemyError as e:
-            raise TemplateSqlError("finding", name, str(e))
+            raise TemplateSqlError(e, operation="find", name=name)
```

Comment:
Does "find" read better than "finding"?

Reply:
I decided not to attempt to construct beautiful sentences, and try to represent the context a bit more concretely by using method names. I almost backtracked on that at least once, but shook it off. A lot of the operations just didn't verb nicely and I tried to go for some consistency. But perhaps I should have reworded the template string to fit that more cleanly.